frontend/src/repository/exams.js

```diff
@@ -2,18 +2,18 @@ import axios from 'axios';
 import moment from "moment";
 
 export const getExams = async () => {
-    const { data } = await axios.get('http://localhost:4000/api/exams');
+    const { data } = await axios.get('/api/exams');
```

Comment:
As mentioned @ValentinChion , if you can revert the changes on this file to avoid conflicts.